docs/content/en/docs/getting-started/ports/_index.md

```diff
@@ -1,20 +1,47 @@
 ---
-title: "3. Ports and protocols"
-linkTitle: "3. Ports/protocols"
+title: "3. Cluster Networking"
+linkTitle: "3. Cluster Networking"
 weight: 16
 aliases:
     /docs/reference/ports/
 description: >
-  Ports used with an EKS Anywhere cluster
+  EKS Anywhere cluster networking
 ---
 
+#### Cluster Networking
+EKS Anywhere clusters use the `clusterNetwork` field in the cluster spec to allocate pod and service IPs. Once the cluster is created, the `pods.cidrBlocks`, `services.cidrBlocks` and `nodes.cidrMaskSize` fields are immutable. As a result, extra care should be taken to ensure that there are sufficient IPs and IP blocks available when provisioning large clusters.
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: my-cluster-name
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - 192.168.0.0/16
+    services:
+      cidrBlocks:
+      - 10.96.0.0/12
+```
+
+The cluster `pods.cidrBlocks` is subdivided between nodes with a default block of size `/24` per node, which can also be [configured via]({{< relref "../optional/cni/#node-ips-configuration-option" >}}) the  `nodes.cidrMaskSize` field. This node CIDR block is then used to assign pod IPs on the node.
+
+{{% alert title="Warning" color="warning" %}}
+The maximum number of nodes will be limited to the number of subnets of size `/24` (or `nodes.cidrMaskSize` if configured) that can fit in the cluster `pods.cidrBlocks`.
+
+The maximum number of pods per node is also limited by the size of the node CIDR block. For example with the default `/24` node CIDR mask size, there are a maximum of 256 IPs available for pods. Kubernetes recommends [no more than 110 pods per node.](https://kubernetes.io/docs/setup/best-practices/cluster-large/)
+{{% /alert %}}
+
+#### Ports and Protocols
+
 EKS Anywhere requires that various ports on control plane and worker nodes be open.
 Some Kubernetes-specific ports need open access only from other Kubernetes nodes, while others are exposed externally.
 Beyond Kubernetes ports, someone managing an EKS Anywhere cluster must also have external access to ports on the underlying EKS Anywhere provider (such as VMware) and to external tooling (such as Jenkins).
 
 If you are responsible for network firewall rules between nodes on your EKS Anywhere clusters, the following tables describe both Kubernetes and EKS Anywhere-specific ports you should be aware of.
 
-## Kubernetes control plane
+##### Kubernetes control plane
```

Comment:
```suggestion
### Kubernetes control plane
```
I suggested changing the heading above to ##. If you agree, then this should be ###.